pkg/util/utils.go

```diff
@@ -101,3 +107,34 @@ func (m MultiError) ToError() error {
 	}
 	return fmt.Errorf(strings.Join(errStrings, "\n"))
 }
+
+func ParseOcHelpCommand(cmdOut []byte) []string {
+	ocOptions := []string{}
+	ocOptionRegex := regexp.MustCompile(`(?s)Options(.*)OpenShift images`)
+	matches := ocOptionRegex.FindSubmatch(cmdOut)
+	if matches != nil {
+		tmpOptionsList := string(matches[0])
+		for _, value := range strings.Split(tmpOptionsList, "\n")[1:] {
+			tmpOption := strings.Split(strings.Split(strings.TrimSpace(value), "=")[0], "--")
+			if len(tmpOption) > 1 {
+				if tmpOption[1] == "env" {
```

Comment:
this is not so good imo. This creates a dependency to the start command. Instead of hard coding this, this method should take a map with the existing mappings (basically _minishiftToClusterUp_) and do this dynamically. This way, we can for example add another option re-map to the start command and all will continue to work.